packages/spec/package.json

```diff
@@ -37,6 +37,8 @@
     "@types/styled-components": "^5.1.8",
     "create-hops-app": "15.0.0-nightly.1",
     "cross-fetch": "^3.0.4",
+    "esbuild-jest": "^0.5.0",
+    "esbuild-loader": "^2.10.0",
```

Comment:
I would still add the `esbuild` dependency here as well as `esbuild-loader` might get fewer releases than `esbuild` and otherwise we would only get `esbuild` updates via the `yarn.lock` file maintenance - which feels wrong to me.

Reply:
I checked it & `esbuild-loader` uses the `esbuild` it brings with it. So there's no use in adding your own `esbuild`. Also the `esbuild`, that's a transient dependency of `esbuild-loader` is guaranteed to work. Which might not be the case with the `esbuild` you add yourself to the project.

Reply:
Ok, you are right. We are only using/importing `esbuild-loader`. So it makes sense then. 👍